### PR TITLE
IMP tax_balance: breadcrumb name

### PR DESCRIPTION
**Pull request by bealdav**
Without this code it only display `Taxes Balances` in breadcrumb

tax_bal

cc @eLBati @sbidoul @antespi @astirpe @alexis-via

### CODE REVIEW

**Review by eLBati — APPROVED** [review]
Thanks! Could you just increase version following https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers ?

File: account_tax_balance/wizard/open_tax_balances.py [hunk 1]
```diff
@@ -31,7 +31,23 @@ def onchange_date_range_id(self):
...
+            [('code', '=', self.env.user.lang or 'en_US')])
+        date_format = lang and lang.date_format or "%m/%d/%Y"
+        infos = {'name': vals['name'], 'target': _(self.target_move),
+                 'from': self.from_date.strftime(date_format),
+                 'to': self.to_date.strftime(date_format),
+                 'company': self.company_id.name}
+        # name of action which is displayed in breacrumb
+        vals["name"] = _(
+            "%(name)s: %(target)s from %(from)s to %(to)s"
+            " (%(company)s)") % infos
```
Comment (astirpe) [inline account_tax_balance/wizard/open_tax_balances.py]: IMO the company should be displayed only in case the user belongs to the "Multi Companies" group.

**Review by astirpe — APPROVED** [review]
:+1: 
Tested locally, since Runbot is not available.

**Comment by bealdav** [comment]
Thanks for your review

File: account_tax_balance/wizard/open_tax_balances.py [hunk 2]
```diff
@@ -31,7 +31,25 @@ def onchange_date_range_id(self):
     def open_taxes(self):
         self.ensure_one()
         action = self.env.ref('account_tax_balance.action_tax_balances_tree')
-        vals = action.read()[0]
+        act_vals = action.read()[0]
+        # override action name doesn't work in v12 or v10
+        # we need to build a dynamic action on main keys
+        vals = {x: act_vals[x] for x in act_vals
```
Comment (pedrobaeza) [inline account_tax_balance/wizard/open_tax_balances.py:37]: No need to make this filtering

Comment (bealdav) [reply account_tax_balance/wizard/open_tax_balances.py:37]: @pedrobaeza  If I don't do that it doesn't work, sorry ;-)

Comment (pedrobaeza) [reply account_tax_balance/wizard/open_tax_balances.py:37]: That's very weird, as previous code was working also with a raw read...

Comment (bealdav) [reply account_tax_balance/wizard/open_tax_balances.py:37]: In previous code there was no override of the action name.

In returned vals the override name is transmit to all args of the stack but the problem comes from _compute_display_name() which compute another value than overrided name

Comment (pedrobaeza) [reply account_tax_balance/wizard/open_tax_balances.py:37]: OK, let's go this way.

**Review by pedrobaeza — APPROVED** [review]

File: account_tax_balance/wizard/open_tax_balances.py (hunk 2 above)
Comment (bealdav) [reply account_tax_balance/wizard/open_tax_balances.py:37]: Don't hesitate to try both code to see if I missed smth or not if any doubt 

Thanks Pedro

Comment (pedrobaeza) [reply account_tax_balance/wizard/open_tax_balances.py:37]: And thanks for the explanation :)

**Pull request merged**